.github/workflows/publish.yml

```diff
@@ -0,0 +1,49 @@
+name: publish
+
+on:
+  push:
+    tags:
+    - v*
```

Comment:
We don’t currently use the `v` prefix. Would it be possible to continue the current naming?

Reply:
Done!